docs/getting-started/01-installation.md

```diff
@@ -0,0 +1,36 @@
+# Installation
+
+For many, the easiest way to install Prism is as a node module. 
+
+```bash
+npm install -g @stoplight/prism-cli
+# or
+yarn global add @stoplight/prism-cli
+```
+
+## Executable Binaries
+
+For users without Node and/or NPM/Yarn, we provide standalone packages for [all major platforms](https://github.com/stoplightio/prism/releases). The quickest way to install the appropriate package for your operating system is via this shell script:
+
+```bash
+curl -L https://raw.githack.com/stoplightio/prism/master/install | sh
+```
+
+<!-- theme: info -->
+> The binaries do _not_ auto-update, so you will need to run it again to install new versions.
+
+## Docker
+
+Prism is available as a Docker image. We recommend specifying the major version you'd like to use as a tag:
+
+```bash
+docker run -P stoplight/prism:3 mock -h 0.0.0.0 api.oas2.yml
```

Comment:
It is important use the `--init` flat to inject the `tini` process manager, otherwise the SIGINT command won't be handled correctly — and it's important for CLI images

```suggestion
docker run --init -P stoplight/prism:3 mock -h 0.0.0.0 api.oas2.yml
```